src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,24 @@
-export const TodoFilter = () => (
+interface Props {
+  filter: string,
+  inputQuery: string,
+  onFilter: (filter: string) => void,
+  onInputQuery: (inputQuery: string) => void,
+}
+
+export const TodoFilter = ({
+  filter,
+  inputQuery,
+  onFilter,
+  onInputQuery,
+}: Props) => (
   <form className="field has-addons">
```

Comment:
if user press Enter in input, the page will be reloaded. Better to add onSubmit event prevent default.